bin/checkver.ps1

```diff
@@ -105,6 +107,7 @@ $Queue | ForEach-Object {
     Register-ObjectEvent $wc downloadstringcompleted -ErrorAction Stop | Out-Null
 
     $githubRegex = '\/releases\/tag\/(?:v|V)?([\d.]+)'
+    $chocoRegex = 'Chocolatey.*?([\d.]+)[\r\n]'
```

Comment:
The regex is too ambiguous (See: https://regex101.com/r/QUv1Em/1)


Better would be `Nuspec reference:\s*&lt;version&gt;(.*)&lt;\/version&gt;`
(See: https://regex101.com/r/QUv1Em/2)

Reply:
Use `>([\d.]+)<\/h2>` as the regex.

Reply:
Or `<h2[^>]+>([\d.]+)<\/h2>` 😄

Reply:
`>([\d.]+)<\/h2>` is enough, I thought, with only 1 found, that is the version number:)

Reply:
Okay, changed it.